### DISCUSSION TITLE
(Suggestion) SeaWeed and it's potential uses

### DISCUSSION THREAD

**Discussion by KnightofChivalry** [cataclysmbn/Cataclysm-BN#5285] [General]
After a discord discussion I was suggested to post the suggestion here.

The suggestion is to add seaweed. from the discussion we discerned that (so far) 2 things could be made with seaweed and that is Roman glass and a rudimentary form of plastic. I'm sure there are more uses for seaweed that could be figured out down the line but still. (This would be more beneficial for innawoods players)

How it spawns could simply be in deep-water to keep it simple, Can also do some aqua-farming with seaweed too as I was told the code for aqua farming already exists with cattails.

So seaweed can have a sort of niche with applied science and other skills such as cooking.

(Ignore this as someone else made a much better post saying the same thing)

**Comment by scarf005** [comment]
> (Ignore this as someone else made a much better post saying the same thing)

thanks for opening this discussion, but as mentioned above, let's continue in #5286
